SDKs/python_sdk/import_products_from_shopify_csv/import_products.py

```diff
@@ -0,0 +1,131 @@
+import csv
+import sys
+
+from builton_sdk import Builton
+from settings import CSV_DELIMITER, API_KEY, BEARER_TOKEN
+from termcolor import cprint
+
+"""
+    MAPPING: Shopify and BuiltOn Product
+
+        Shopify         |           BuiltOn
+=======================================================
+    Title               |   name
+    Body (HTML)         |   description
+    Handle              |   external_reference
+    Vendor              |   properties['vendor']
+    Type                |   tags
+    Tags                |   tags
+    Published           |   active
+    Variant Price       |   price
+    Variant Barecode    |   properties['barecode`]
+    Image Src           |   image_url
+    Variant Grams       |   properties['grams']
+    Variant Weight Unit |   properties['weight_unit']
+=======================================================
+    Info: The field `properties` is a dictionary. You can fill it as you want, and the keys `grams`,
+     `vendor` etc are not set in stone. You can change them as you like.
+    
+"""
+
+
+def main(api_key=None, bearer_token=None):
+    api_key = api_key if api_key is not None else API_KEY
+    bearer_token = bearer_token if bearer_token is not None else BEARER_TOKEN
+
+    builton = Builton(api_key=api_key, bearer_token=bearer_token)
+    cprint("BuiltOn skd is ready!", "cyan")
+
+    import_products(builton)
+
+
+def import_products(builton: Builton):
+    with open('products_export.csv', newline='') as csv_product_file:
+        csv_reader = csv.DictReader(csv_product_file, delimiter=CSV_DELIMITER)
+        for shopify_product in csv_reader:
+            cprint("============================================================")
+            try:
+                cprint("======================== Properties ========================")
+                vendor = shopify_product.get('Vendor')
+                cprint("Vendor: %s" % vendor, "cyan")
+                grams = shopify_product.get('Variant Grams')
+                cprint("Grams: %s" % grams, "cyan")
+                weight_unit = shopify_product.get('Variant Weight Unit')
+                cprint("Weight Unit: %s" % weight_unit, "cyan")
+
+                properties = {
+                    'vendor': vendor,
+                    'grams': grams,
+                    'weight_unit': weight_unit
+                }
+
+                cprint("=========================== Tags ===========================")
+                tags = [tag for tag in shopify_product.get('Tags').split(',')]
```

Comment:
Suggestion:
`tags = shopify_product.get('Tags').split(',')`

Reply:
hmmm :thinking: it should work! thx :)